src/components/Button/Button.scss

```diff
@@ -73,11 +73,11 @@
     }
   }
   .monday-style-button--left-icon {
-    padding-right: $spacing-small;
+    margin-right: var(--spacing-small);
```

Comment:
because padding effect the actual size of the button icon (and then "24px" is not really 24px)